packages/next-bundle-analyzer/index.js

```diff
@@ -0,0 +1,21 @@
+module.exports = pluginOptions => (nextConfig = {}) => {
+  return Object.assign({}, nextConfig, {
+    webpack (config, options) {
+      const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer')
```

Comment:
Should this be inside the following if statement?

Reply:
Yeah, good catch